recipes/09/09_group_by_session_window.md

```diff
@@ -0,0 +1,57 @@
+# 09 Aggregating Time Series Data (Session Windows)
```

Comment:
Did you consider calling the recipe something along the lines of "Analyze Sessions in Time Series Data" or something with "Sessionization"?

Reply:
Wanted to make it coherent with the existing one about windowing, but I can rename it.